zulip/zulip/__init__.py

```diff
@@ -708,6 +708,20 @@ def get_messages(self, message_filters):
             request=message_filters
         )
 
+    def get_messages_matching_narrow(self, request):
+        # type: (Dict[str, Any]) -> Dict[str, Any]
+        '''
+            Example usage:
+
+            >>> client.get_messages_matching_narrow(request)
```

Comment:
I think we better provide an example narrow input in `request` rather than a variable name, to explain how to use this.  Might be worth commenting that the format is the same as for `get_messages`.